pypesto/ensemble/ensemble.py

```diff
@@ -256,6 +321,37 @@ def _compute_summary(tmp_array, percentiles_list):
         # also return the object
         return self.prediction_summary
 
+    def compute_chi2(self, amici_objective: AmiciObjective):
+        if (self.prediction_summary[MEAN] is None) or (
+                self.prediction_summary[WEIGHTED_SIGMA] is None):
+            try:
+                self.compute_summary(
+                        weighting=True,
+                        compute_weighted_sigma=True)
+            except TypeError:
+                raise ValueError('Computing a summary failed.')
+        n_conditions = len(self.prediction_results[0].conditions)
```

Comment:
Can conditions differ between predictions? If so, can simply raise a `NotImplementedError` if there are different conditions between the prediction results.

Reply:
they should not be able to, as the ensembles currently are created from the same amici_model which should generate the prediction the same no matter what the actual parameter values are.

Might be interesting way down the train, when we allow for different model structures to contribute to the same ensemble.

Reply:
Conditions are provided via `amici.ExpData` (a single `amici.Model` can simulate different sets of conditions).